doc/nrf/glossary.rst

```diff
@@ -455,7 +491,7 @@ Glossary
       A timer that causes a system reset if it is not poked periodically.
 
    West
-      A multiple repository management system used by Zephyr and the |NCS|.
+      A multiple-repositories management system used by Zephyr and the |NCS|.
```

Comment:
Is this correct? Shouldn't it stay singular?

Reply:
It turned out that it is too long anyway. Fixed differently.